packages/rollup-plugin-html/src/output/injectServiceWorkerRegistration.ts

```diff
@@ -0,0 +1,47 @@
+import path from 'path';
+import { Document } from 'parse5';
+import {
+  findElement,
+  getTagName,
+  appendChild,
+  createScript,
+  setAttribute,
+} from '@web/parse5-utils';
+
+export interface injectServiceWorkerRegistrationArgs {
+  document: Document;
+  serviceWorkerPath: string;
+  outputDir: string;
+  htmlFileName: string;
+}
+
+export function injectServiceWorkerRegistration(args: injectServiceWorkerRegistrationArgs) {
+  const { document, serviceWorkerPath, outputDir, htmlFileName } = args;
+  const body = findElement(document, e => getTagName(e) === 'body');
+  if (!body) {
+    throw new Error('Missing body in HTML document.');
+  }
+
+  let serviceWorkerUrl = serviceWorkerPath.replace(`${outputDir}${path.sep}`, '');
```

Comment:
```suggestion
  let serviceWorkerUrl = path.relative(outputDir, serviceWorkerUrl);
```

This would fail when `outputDir` has a trailing slash. I *think* this has the same but leaves handling edge cases to the path library.

Reply:
indeed - applied 👍 
thx 🙇‍♂️